assets/js/frontend/course-video/disable-complete-lesson-button.js

```diff
@@ -0,0 +1,5 @@
+( () => {
+	document.querySelector(
```

Comment:
I think I reviewed this PR as part of the other one. haha

So I'm just linking the important comments related to this PR:
- https://github.com/Automattic/sensei/pull/4546/files#r774178891
- https://github.com/Automattic/sensei/pull/4546#pullrequestreview-838833100